scripts/st2bootstrap-deb.template.sh

```diff
@@ -111,31 +111,6 @@ setup_args() {
     fi
   fi
 
-  # Python 3.6 package is not available in Ubuntu Xenial
-  # Installer can add it via 3rd party PPA based on user agreement
-  if [[ "$SUBTYPE" = 'xenial' ]]; then
```

Comment:
We could leave this if here and just print an error that Xenial is not supported since v3.5.0 in case someone still tries to run new installer script on Xenial.

Reply:
Thoughts on this comment... @Kami @armab @blag 

I don't mind either way - but we'd have to  put quite a few different - if "xenial" do this checks back in. Plus the 3.5 and "unstable" xenial packages are no-longer built daily - so what they'd get is an old version anyway.

As I understand the install process then if they install 3.4 xenial they will get the "master" st2_bootstrap - but they'd get the st2bootstrap-deb.sh from the 3.4 branch - and so would have all the stuff there.

So to me, it's safer to just remove the ifs, and say that we only support bionic/focal etc.